### ISSUE TITLE
Anzeige in IOS Safari mit Fehler - in IOS Chrome alles ok ( evtl durch letztes IOS Update auf 16.4 )

### ISSUE THREAD

**Issue by Uwe-Kaki** [Zefau/ioBroker.jarvis#2308]
Hallo,
ich habe Jarvis seit Monaten im Einsatz, funktionierte bisher auf WIndows / IPad und iPhone problemlos. Auf Windows ist auch jetzt noch alles ok. Selbst "Safari" Shortcuts auf den HomeScreen haben funktioniert, nun aber geht es nicht mehr.

Auf iPhone / iPad habe ich jetzt folgendes "Phänomen" :

Die HomeScreen Shortcuts funktionieren nicht mehr, auch der Aufruf über den Jarvis Adapter liefert eine Fehlermehldung nach einiger Zeit
So rufe ich die Jarvis Visualisierung auf :

**Screenshots**
![1680336687365-jarvis-1](https://user-images.githubusercontent.com/75257707/229339887-57cf5f2d-89a4-4249-a902-7a8dcdd2d2d9.jpg)

Dann erscheinen nach einer Weile diese Fehlermeldungen.

![1680336787952-jarvis-2-resized](https://user-images.githubusercontent.com/75257707/229339904-578e2d28-2735-471d-a0a8-508a76a466b2.jpg)

Wenn ich nun auf den "Kompass" rechts oben klicke, dann öffnet sich das Ganze in **IOS-Chrome** und alles ist so wie erwartet, das bedeutet der Jarvis Adapter selber funktioniert mit allen Anzeigen, nur eben nicht unter Safari.

![1680336864465-jarvis-3-resized](https://user-images.githubusercontent.com/75257707/229339914-c2b239d9-974b-4525-b54a-9493abc2e574.jpg)

Vorher hat alles in Safari funktioniert und zwar direkt und ohne Umwege. Für die Anzeige war das völlig ausreichend.

Ich kann nicht genau sagen, ab wann es nicht mehr funktioniert, da ich längere Zeit nicht daheim war und die Visualisierung remote nicht verwendet habe.

Probiert habe ich Neustart aller Geräte, Webseite Verlauf Löschen in Safari - brachte alles keinen Erfolg.

**Comment by Zefau** [comment]
Passiert das auch, wenn du jarvis direkt im Safari aufrufst?

**Comment by Uwe-Kaki** [comment]
Ja, deswegen bin ich ja drauf gekommen, weil ich auf dem IPad Homescreen die Safari-Verknüpfung abgelegt hatte ( bei iPhone das gleiche ) um nicht immer über ioBroker einsteigen zu müssen.Beispiel : ( alles auf dem iPad )- in Chrome direkt eingegeben https://192.168.178.30:8082/jarvis/index.html#/0 klapptIn Safari direkt eingegeben https://192.168.178.30:8082/jarvis/index.html#/0 passiert folgendes nach einer Weile Ab und zu ( nicht direkt reproduzierbar ) bekomme ich bei Safari den Hinweis : Webseite unsicher, bitte Zertifikat überprüfen, wenn dir das Risiko bewusst ist, dann verbinde trotzdem zur unsicheren Seite ) es geht dann aber trotzdem nicht.Seit dem letzten IOS Update funktioniert es bei mir unter Safari nicht mehr ( gefühlt, da ich zu dem Zeitpunkt unterwegs war und nur über VPN auf Jarvis zugreifen konnte )Am Windows PC bzw Chrome auf iPad / iPhone alles bestens.LG aus dem Norden / best regardsUwe KnöfelAm 16.04.2023 um 16:07 schrieb Zefau ***@***.***>:﻿
Passiert das auch, wenn du jarvis direkt im Safari aufrufst?

**Comment by Uwe-Kaki** [comment]
Hallo,
Ich konnte jetzt die Meldungen mit den Warnhinweisen nachvollziehen und habe Screenshots gemacht.
Dabei ist mir aufgefallen, das die Meldungen sich auf die Slave-IOBroker instance beziehen, Jarvis läuft aber auf dem Master-IOBroker.

soll ich die Screenshots hier einstellen oder wird das zuviel ?

**Comment by Zefau** [comment]
@Uwe-Kaki ist das hier mit v3.2 noch aktuell?
